ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -276,6 +276,16 @@ def NthElement(*, input, n, reverse=False, name="NthElement"):
     return ivy.astype(ivy.sort(input, descending=reverse)[..., n], input.dtype)
 
 
+@to_ivy_arrays_and_back
+def OneHot(*, indices, depth, on_value, off_value, axis=-1, name="OneHot"):
+    return ivy.one_hot(
+        indices, depth, on_value=on_value, off_value=off_value, axis=axis
+    )
+
+
+OneHot.supported_dtypes = ("int32", "int64", "uint8")
```

Comment:
we no longer use this, we now have a decorator for that: https://lets-unify.ai/ivy/deep_dive/data_types.html#supported-and-unsupported-data-types